### PR TITLE
Update main for AWS deployment

### PR DESCRIPTION
**Pull request by danieljaffe**
General code review before AWS deployment of main.

### CODE REVIEW

File: server/src/graphql/api.ts [hunk 1]
```diff
@@ -66,24 +69,25 @@ export const graphqlRoot: Resolvers<Context> = {
...
+    //  check(await listeningSession.save())
 
-      //incrementing length of listeningSession.queueLength
-      const entityManager = getManager();
-      check(await entityManager.increment(ListeningSession, { id: listeningSessionId }, "queueLength", 1))
+    //check this below, is this creating a race condition???
+    //  incrementing length of listeningSession.queueLength
+     const entityManager = getManager();
+     // change this to saving the entity above???
+     check(await entityManager.increment(ListeningSession, { id: listeningSessionId }, "queueLength", 1))
```
Comment (dbreon88) [inline server/src/graphql/api.ts:90]: this may lead to a race condition at high loads, will look into this for future pushes

**Review by dbreon88 — APPROVED** [review]
merge to main looks good

File: server/src/graphql/api.ts [hunk 2]
```diff
@@ -31,7 +32,9 @@ export const graphqlRoot: Resolvers<Context> = {
   Query: {
     self: (_, args, ctx) => ctx.user,
     survey: async (_, { surveyId }) => (await Survey.findOne({ where: { id: surveyId } })) || null,
-    listeningSession: async (_, { sessionId }) => (await ListeningSession.findOne({ where: { id: sessionId }, relations: ['queue', 'owner', 'partyRockers'] })) || null,
+    listeningSession: async (_, { sessionId }) => { const result = await ListeningSession.findOne({ where: { id: sessionId }, relations: ['queue', 'owner', 'partyRockers'] })
+    console.log("result", result)
```
Comment (aybeshlikyan) [inline server/src/graphql/api.ts]: comment out the console.log

File: web/src/view/page/HomePage.tsx [hunk 3]
```diff
@@ -1,26 +1,170 @@
...
+  //const [getSession, { loading,data }] = useLazyQuery<FetchListeningSession, FetchListeningSessionVariables>(fetchListeningSession, { variables: { sessionId: sessionID}})
+  const client = useApolloClient()
+
+  //@ts-ignore
+  const handleChange = async event => {
+    await setSessionID(parseInt(event.target.value));
+  };
+
+	async function joinSession(event: React.MouseEvent): Promise<void> {
+    console.log("in join session")
```
Comment (aybeshlikyan) [inline web/src/view/page/HomePage.tsx]: remove console.log

File: web/src/view/page/HomePage.tsx [hunk 4]
```diff
@@ -1,26 +1,170 @@
...
+    event.preventDefault();
+
+  console.log("sessionID from state", sessionID)
```
Comment (aybeshlikyan) [inline web/src/view/page/HomePage.tsx]: remove console.log

File: web/src/view/page/HomePage.tsx [hunk 5]
```diff
@@ -1,26 +1,170 @@
...
+    if((data?.listeningSession?.id) == undefined){
+      alert("Failed Fetch Session");
+      return;
+    }
+    else {
+     idSession = data?.listeningSession?.id;
+    }
+
+
+    console.log("data.listeningsession", data.listeningSession)
```
Comment (aybeshlikyan) [inline web/src/view/page/HomePage.tsx]: remove console.log

File: web/src/view/page/LecturesPage.tsx [hunk 6]
```diff
@@ -1,22 +1,103 @@
...
+    backgroundColor: theme.palette.background.paper,
+  },
+  title: {
+    margin: theme.spacing(4, 0, 2),
+  },
+}))
+
 export function LecturesPage(props: LecturesPageProps) {
-  const [sessions] = useState(["Session 1", "Session 2", "Session 3"]);
+  console.log('LecturesPages')
```
Comment (aybeshlikyan) [inline web/src/view/page/LecturesPage.tsx]: remove console.log

File: web/src/view/page/LecturesPage.tsx [hunk 7]
```diff
@@ -1,22 +1,103 @@
...
+  const idSession = Number(props.sessionId)
+  console.log('ID session from lecturePage')
+  console.log(idSession)
```
Comment (aybeshlikyan) [inline web/src/view/page/LecturesPage.tsx]: comment out these console.logs

File: web/src/view/page/LecturesPage.tsx [hunk 8]
```diff
@@ -1,22 +1,103 @@
...
+  const { loading: loadingSongs, data: dataSongs } = useQuery<FetchSongs>(fetchSongs)
+  const { loading: loadingSession, data: sessionData } = useQuery<
+    FetchListeningSession,
+    FetchListeningSessionVariables
+  >(fetchListeningSession, {
+    variables: { sessionId: idSession },
+  })
+
+  console.log(dataSongs)
+  console.log(sessionData)
```
Comment (aybeshlikyan) [inline web/src/view/page/LecturesPage.tsx]: comment out console.log

File: web/src/view/page/LecturesPage.tsx [hunk 9]
```diff
@@ -1,22 +1,103 @@
...
+            Songs
+          </Typography>
+          <div className={classes.demo}>
+            <List dense={dense}>
+              {dataSongs.songs.map(currSong => (
+                <ListItem>
+                  <ListItemIcon>
+                    <AddIcon
+                      onClick={() => {
+                        console.log(currSong)
```
Comment (aybeshlikyan) [inline web/src/view/page/LecturesPage.tsx]: comment out console.log

File: web/src/view/playground/Surveys.tsx [hunk 10]
```diff
@@ -40,6 +40,7 @@ function SurveyList() {
   if (!data || data.surveys.length === 0) {
     return <div>no surveys</div>
   }
+  console.log(data)
```
Comment (aybeshlikyan) [inline web/src/view/playground/Surveys.tsx]: comment out console.log

**Review by aybeshlikyan — CHANGES_REQUESTED** [review]
Looks good! I commented on all the console.logs that should be commented out before we merge.

**Review by danieljaffe — COMMENTED** [review]
All console.log in these 25 files have been commented out. Dylan's comment not resolved for this merge.

**Pull request merged**